Arithmetic_Progression/ArithmeticProgression.php

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Fix file name to ` Arithmetic_Progression.php`

Reply:
Done!